processor/elasticinframetricsprocessor/processor_test.go

```diff
@@ -111,6 +111,85 @@ func TestProcessMetrics(t *testing.T) {
 	}
 }
 
+func TestDropOriginalMetrics(t *testing.T) {
+	testCases := []struct {
+		name                   string
+		cfg                    *Config
+		createMetrics          func() pmetric.Metrics
+		expectedPodname        string
+		expectedMetricName     string
+		expetedMetricValue     float64
+		expetedLengthOfMetrics int
+	}{
+		{
+			name: "ProcessMetrics when AddK8sMetrics is enabled and DropOriginal is enabled",
+			cfg:  &Config{AddK8sMetrics: true, DropOriginal: true},
+			createMetrics: func() pmetric.Metrics {
+				md := pmetric.NewMetrics()
+				rm := md.ResourceMetrics().AppendEmpty()
+				rm.Resource().Attributes().PutStr("k8s.pod.name", "test-pod")
+				sm := rm.ScopeMetrics().AppendEmpty()
+				// Creating first metric
+				metric1 := sm.Metrics().AppendEmpty()
+				metric1.SetName("k8s.pod.cpu_limit_utilization")
+				dp1 := metric1.SetEmptyGauge().DataPoints().AppendEmpty()
+				dp1.Attributes().PutBool("otel_remapped", true)
```

Comment:
Shouldn't this attribute be added by opentelemetry-lib?

Reply:
Indeed was not needed. I removed it